### PR TITLE
EOS-25820: Adopting to new message bus changes 

### PR DESCRIPTION
**Pull request by vimalraghwani20**
# Problem Statement
- Adopting to new message bus changes 

# Design
-  For Bug, Describe the fix here.
-  For Feature, Post the link for design

# Coding
   Checklist for Author
-  [ ] Coding conventions are followed and code is consistent

# Testing 
  Checklist for Author
- [ ] Unit and System Tests are added
- [ ] Test Cases cover Happy Path, Non-Happy Path and Scalability
- [ ] Testing was performed with RPM

# Impact Analysis
  Checklist for Author/Reviewer/GateKeeper
- [ ] Interface change (if any) are documented
- [ ] Side effects on other features (deployment/upgrade)
- [ ] Dependencies on other component(s)

# Review Checklist 
  Checklist for Author
- [ ] JIRA number/GitHub Issue added to PR
- [ ] PR is self reviewed
- [ ] Jira and state/status is updated and JIRA is updated with PR link
- [ ] Check if the description is clear and explained

# Documentation
  Checklist for Author
- [ ] Changes done to WIKI / Confluence page / Quick Start Guide

### CODE REVIEW

File: scripts/provisioning/configcmd.py [hunk 1]
```diff
@@ -412,6 +412,11 @@ def configure_s3_schema(self):
   def create_topic(self, admin_id: str, topic_name:str, partitions: int):
     """create topic for background delete services."""
     try:
+      self.logger.info("Init Msg Bus started")
+      endpoints_val = self.get_confvalue_with_defaults("CONFIG>CONFSTORE_S3_KAFKA_ENDPOINTS")
+      self.logger.info(f"Msg Bus end point value is {endpoints_val}")
```
Comment (nileshgovande) [inline scripts/provisioning/configcmd.py:417]: Need to fix

File: scripts/provisioning/s3_start [hunk 2]
```diff
@@ -178,6 +180,15 @@ def main():
       # Uncomment below two lines if you want return bgdelete schedule timings.
       # os.system("sed -i -e \"s,scheduler_schedule_interval:.*,scheduler_schedule_interval:\ 120,\" /etc/cortx/s3/s3backgrounddelete/config.yaml")
       # os.system("sed -i -e \"s,leak_processing_delay_in_mins:.*,leak_processing_delay_in_mins:\ 2,\" /etc/cortx/s3/s3backgrounddelete/config.yaml")
+
+      # Initialize Msg Bus
+      logger.info("Init Msg Bus started")
```
Comment (nileshgovande) [inline scripts/provisioning/s3_start:185]: Need to fix

**Comment by nileshgovande** [comment]
Need to revert/take care of ansible/files/yum.repos.d/centos7.9.2009/cortx.repo before merging

**Comment by shahidSTX** [comment]
Test this with utils branch - `utils_no_conf`

File: scripts/provisioning/configcmd.py (hunk 1 above)
Comment (shahidSTX) [reply scripts/provisioning/configcmd.py:417]: Done.

File: scripts/provisioning/s3_start (hunk 2 above)
Comment (shahidSTX) [reply scripts/provisioning/s3_start:185]: Done.

**Review by shahidSTX — APPROVED** [review]

**Comment by shahidSTX** [comment]
Pre-merge job passed -
image
Deployment successfully done via custom build -
image

**Comment by shahidSTX** [comment]
> Need to revert/take care of ansible/files/yum.repos.d/centos7.9.2009/cortx.repo before merging

Done.

**Pull request merged**